validphys2/src/validphys/pdfbases.py

```diff
@@ -621,6 +623,17 @@ def f_(transform_func):
 LUX_FLAVOUR = copy.deepcopy(FLAVOUR)
 LUX_FLAVOUR.default_elements=('u', 'ubar', 'd', 'dbar', 's', 'sbar', 'c', 'cbar', 'g', 'photon')
 
+EVOL_POL = LinearBasis.from_mapping({
+    r'\Sigma'  : {'u': 1, 'ubar': 1, 'd': 1, 'dbar': 1, 's': 1, 'sbar': 1, 'c': 1, 'cbar': 1 ,'b':1, 'bbar': 1, 't': 1, 'tbar': 1},
```

Comment:
I have a question, for my uderstanding:
I'm not 100% sure I understand this. At the fitting scale singlet should contain only `u,d,s`, is this what is happening here?

Reply:
Ah, let me double check the appropriate expressions here.

Reply:
This part is just used to rotate a given PDF in the flavor basis into `EVOL_POL`. This is not used in the fit but rather in the report to show the result in the `EVOL_POL` basis, so if $t,b$ are zero from the fits then they will not contribute into $\Sigma$.